docs/beta/infrequent-dashboard-support.md

```diff
@@ -48,3 +43,7 @@ The dashboard presents the following information for the currently selected time
 * **Top 50 Expensive Queries**. The top 50 expensive queries set up on Dashboards to run against the Infrequent tier.
 
 You can download the JSON for the dashboard [here](https://sumologic-app-data.s3.amazonaws.com/Infrequent_Dashboard.json). For information about importing dashboards and other content see [Import Content in the Library](/docs/get-started/library#import-content).
+
+## Limitation
+
+* Auto-refesh is not available for dashboard panels running on Infrequent tier.
```

Comment:
Auto-refresh

Reply:
Can you refresh the page because I already made this change in the file?